ls1intum/Artemis
# `Text exercises`: Fix alignment of management buttons

### Checklist
#### General


#### Client

### Motivation and Context


Fixes #7114 
There was an issue with the exercise management buttons. The "Assessment Dashboard" button was not properly aligned.

### Description

The styling of the "Assessment Management" button is now consistent with the other info-buttons.

### Steps for Testing

Prerequisites:
- 1 Text Exercise 

1. Log in to Artemis
2. Navigate to the management view of the text exercise
3. Make sure that the management buttons are aligned properly

### Review Progress






#### Code Review
#### Manual Tests

### Screenshots


![image](https://github.com/ls1intum/Artemis/assets/129070641/64bc6747-f02f-4641-94ee-607d29ec1046)
